ui/.env.sample

```diff
@@ -0,0 +1,7 @@
+VITE_API_URL=
+VITE_API_USERNAME=
+VITE_API_PASSWORD=
```

Comment:
This is intentionally left blank. The user must consciously put in the values of `.env-api` as instructed, as it's the source of truth, otherwise they will see errors. We don't want placeholders here because they need to be correct values.